### PR TITLE
Added AA rounding to Tabbar

### PR DESCRIPTION
**Pull request by JavaCafe01**
Does anti-aliased corner rounding with the theme variable `theme.tabbar_radius`. To disable, set the variable to 0.

(EDIT: changed var name as that was in the docs for some reason before)

### CODE REVIEW

File: module/tabbed.lua [hunk 1]
```diff
@@ -173,12 +173,27 @@ tabbed.update_tabbar = function(tabobj)
...
+
+        local bar_shape = helpers.shape.rrect(0)
+        if bar.position == "top" then
+            bar_shape = helpers.shape.prrect(beautiful.tabbar_corner_radius,
+                                             true, true, false, false)
+        elseif bar.position == "bottom" then
+            bar_shape = helpers.shape.prrect(beautiful.tabbar_corner_radius,
+                                             false, false, true, true)
+        end
+
```
Comment (Nooo37) [inline module/tabbed.lua:185]: The tabbar position could also be on the left or on the right. I think only `boxes` allows for that currently.

File: module/tabbed.lua [hunk 2]
```diff
@@ -8,9 +8,7 @@ In the function themselves, the same object is refered to as "tabobj" which is w
 you will often see something like: "local tabobj = some_client.bling_tabbed" at the beginning
 of a function.
 
---]]
-
-local awful = require("awful")
+--]] local awful = require("awful")
```
Comment (Nooo37) [inline module/tabbed.lua:11]: That looks very weird but who am I to question the choices of your formatter.

**Review by Nooo37 — CHANGES_REQUESTED** [review]
Add support for tabbar on the left or on the right because the tabbar style `boxes` allows for that and it would be an unnecessary inconsistency considering how easy it would be to add.

File: module/tabbed.lua (hunk 2 above)
Comment (JavaCafe01) [reply module/tabbed.lua:11]: Yea, I just use default settings for my formatter

File: module/tabbed.lua (hunk 1 above)
Comment (JavaCafe01) [reply module/tabbed.lua:186]: forgot that people use vertical titlebars and wibars

Comment (JavaCafe01) [reply module/tabbed.lua:192]: @Nooo37 done

**Comment by JavaCafe01** [comment]
This PR breaks Awesomewm stable somehow. Gonna have to figure it out.

**Comment by JavaCafe01** [comment]
Closing this as I cannot figure out why it breaks stable, gonna reopen once they update lol

**Pull request closed without merging**